lib/email-validation.rb

```diff
@@ -3,8 +3,12 @@
 require 'email-validation/config'
 
 module EmailValidation
-  def self.incorrect_email_message(email)
-    "The email address you have entered (#{email}) is incorrect."
+  def self.invalid_email_message
```

Comment:
Should or should not - I don't know, not sure what are the gem creation rules, not aware of all the contexts and for me it was not worth testing everything and risking it'll break sth. Helper in such a small gem? You mean view helper? Not sure if it's a good idea or even if it's possible here. Constant with dynamic helper method? It doesn't sound good to me. I just followed the way defined before by Kacper.